tests/integration/04.Edge.Tests.ps1

```diff
@@ -660,6 +660,67 @@ Describe "Edge" {
         }
     }
 
+    Context "SSH" {
```

Comment:
Great that you are adding tests, but why are you including ssh tests here?  should be separate PR...

Reply:
Ok
it is already a separate commit... (i see this issue when write my testsuite...)

Reply:
its because you aren't branching from master.  you should use master to track upstream, branch from it for every isolated PR you intend to raise.  For this case, dont worry - _as long as the PR tests pass when we execute the full suite against it_